build_info/libtool.control

```diff
@@ -0,0 +1,7 @@
+Package: GNU libtool
```

Comment:
This would be libtool. `Name:` is a completely separate entry in the control file.